randomScripts/Closure/closureBash.sh

```diff
@@ -1,17 +1,19 @@
 #!/bin/bash
 
-years=("18" "17" "16" "16APV" "RunII")
+output_path="$1"
+lower_bin="$2"
+upper_bin="$3"
 
+years=("18" "17" "16" "16APV" "RunII")
 cases=("Default" "CRTT" "CRDY" "EMu")
-
 types=("total_background" "total_data")
 
 for year in "${years[@]}"; do
     for case in "${cases[@]}"; do
         for type in "${types[@]}"; do
-            command="python closurePlotter.py /eos/user/g/gdecastr/SUEPCoffea_dask/Plots/${case}_2D_${year} ${type} 14 21 220 135 /eos/user/g/gdecastr/SUEPCoffea_dask/CommonClosures/${year}_${case}_${type}"
-            echo $command
-            $command 
+            command="python closurePlotter.py /eos/user/g/gdecastr/SUEPCoffea_dask/Plots/${case}_2D_${year} ${type} ${output_path}/${year}_${case}_${type} ${lower_bin} ${upper_bin}" 
```

Comment:
How are you transforming 4 arguments (in closurePlotter.py) into 2 in bash? Did you test that the script works?

Reply:
Also, now the output file path (for the pdf) is going before the binning while in closurePlotter.py it is still sys.argv[7]. This will crash I think